javascript/advanced_examples/gdx_making_a_chart.html

```diff
@@ -144,34 +135,34 @@
 			}
 		}; 		
     
-    window.onload = function() {
+		window.onload = function() {
 			let ctx = document.getElementById('canvas').getContext('2d');
 			window.myLine = new Chart(ctx, config);
```

Comment:
Is there a reason this needs to be on the window?

Reply:
I am not sure. I can look into it, I just fix the formatting.